packages/calling/calling/package.json

```diff
@@ -43,7 +43,7 @@
     "jest-html-reporters": "3.0.11",
     "platform": "1.3.6",
     "uuid": "8.3.2",
-    "webex": "1.154.1",
+    "webex": "workspace:^",
```

Comment:
Pls revert this.

Reply:
No without this we won't be able to import webex inside calling which is required for initialization